### PR TITLE
Migrate all tests to Jest

### PR DESCRIPTION
**Pull request by colinrotherham**
This PR migrates `@hapi/lab`, `@hapi/code` and `sinon` to [Jest](https://jestjs.io)

### CODE REVIEW

File: src/server/plugins/initialiseSession/helpers.ts [hunk 1]
```diff
@@ -63,6 +63,8 @@ export function verifyToken(decodedToken) {
     })
     return { isValid: true }
   } catch (err) {
+    console.log(err)
```
Comment (alexluckett) [inline src/server/plugins/initialiseSession/helpers.ts:66]: this is already logged further up the chain, can we remove please

File: test/cases/server/utils/verifyToken.test.mjs [hunk 2]
```diff
@@ -1,46 +1,36 @@
...
-    sinon.stub(config, 'initialisedSessionKey').value('something new')
-    expect(verifyToken(decodedToken).isValid).to.be.false()
-
-    sinon.restore()
-    sinon.stub(config, 'initialisedSessionAlgorithm').value('PS384')
-    expect(verifyToken(decodedToken).isValid).to.be.false()
+    jest.replaceProperty(config, 'initialisedSessionKey', 'something new')
+    expect(verifyToken(decodedToken).isValid).toBe(false)
+  })
+  test('can detect incorrect algorithm', () => {
```
Comment (alexluckett) [inline test/cases/server/utils/verifyToken.test.mjs:29]: bonus points for splitting this out :-)

Comment (colinrotherham) [reply test/cases/server/utils/verifyToken.test.mjs:29]: Cheers, needs must. Jest couldn't clear replaced stubbed properties mid-test!

File: src/server/plugins/initialiseSession/helpers.ts (hunk 1 above)
Comment (colinrotherham) [reply src/server/plugins/initialiseSession/helpers.ts:66]: Well spotted, shouldn't have been left in. Will remove

**Review by alexluckett — APPROVED** [review]

**Pull request merged**